e2e/cluster/node_runner.go

```diff
@@ -114,7 +115,7 @@ func (runner *dockerNodeRunner) Run(node *NodeSpec) error {
 
 	logStream := os.Stdout
 	if !runner.logsToStdout {
-		logFilename := filepath.Join(runner.logsDir, fmt.Sprintf("%s-%v.log", runner.logPrefix, node.ID))
+		logFilename := filepath.Join(runner.logsDir, fmt.Sprintf("%s-%v-%d.log", runner.logPrefix, node.ID, rand.Intn(100)))
```

Comment:
Why do we need that random number? node.ID is already a random number. If its not enought, you can make it longer and increase entropy.

Reply:
Jeka, this was because when I crashed the validator, and run another one it was collapsing with the one we have. So I think it is better to have it separated, normally it will only create one. In this case with 2 was easier for me to follow the flow.

Reply:
I get it. Why dont use just an increasing number? I'll make easier to find a proper log.

Reply:
You can reuse atomic counter type for it.

Reply:
you are right, yes